sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4856,16 +4856,16 @@ object functions {
    * @group udf_funcs
    * @since 2.0.0
    */
-  @deprecated("Untyped Scala UDF API is deprecated, please use typed Scala UDF API such as " +
-    "'def udf[RT: TypeTag](f: Function0[RT]): UserDefinedFunction' instead.", "3.0.0")
+  @deprecated("`udf` method with return type parameter is deprecated. " +
```

Comment:
I checked that the java udf API also has return type parameter. Maybe better to say `Scala udf method ...`

Reply:
ok